### PR TITLE
#CNV-39410 - remove web console overview from map

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.16+

Issue: [CNV-39410](https://issues.redhat.com//browse/CNV-39410)

Link to docs preview:

QE review: QE not needed, removing an assembly per docs team decision

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.17` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.16. And any PR going into `main` must also target the latest version branch (enterprise-4.17).

If the update in your PR does NOT apply to version 4.17 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: virt/live_migration/virt-configuring-live-migration.adoc [hunk 1]
```diff
@@ -10,24 +10,14 @@ You can configure live migration settings to ensure that the migration processes
...
 
-[id="live-migration-settings"]
-== Live migration settings
-
-You can configure the following live migration settings:
-
-* xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-live-migration-limits_virt-configuring-live-migration[Limits and timeouts]
-* xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[Maximum number of migrations per node or cluster]
-
-include::modules/virt-configuring-live-migration-limits.adoc[leveloffset=+2]
```
Comment (adellape) [inline virt/live_migration/virt-configuring-live-migration.adoc:21]: Could this file be removed from the repo now or moved to `_unused_topics/`? Not seeing it included anywhere else.

File: virt/monitoring/virt-runbooks.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ toc::[]
 :!virt-runbooks:
 You can use the procedures in these runbooks to diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts].
 
-{VirtProductName} alerts are displayed on the *Virtualization* -> *Overview* -> xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-overview_virt-web-console-overview[*Overview* tab] in the web console.
+{VirtProductName} alerts are displayed in the *Virtualization* -> *Overview* -> tab in the web console.
```
Comment (adellape) [inline virt/monitoring/virt-runbooks.adoc:13]:
```suggestion
{VirtProductName} alerts are displayed in the *Virtualization* -> *Overview* tab in the web console.
```

File: virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc [hunk 3]
```diff
@@ -10,7 +10,7 @@ Red Hat images are xref:../../../virt/virtual_machines/creating_vms_rh/virt-crea
 
 Red Hat images are automatically updated. You can disable and re-enable automatic updates for these images. See xref:../../../virt/storage/virt-automatic-bootsource-updates.adoc#managing-rh-boot-source-updates_virt-automatic-bootsource-updates[Managing Red Hat boot source updates].
 
-Cluster administrators can enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
+Cluster administrators can enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../../virt/about_virt/about-virt.adoc#[web console].
```
Comment (adellape) [inline virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc:13]: Missing anchor:
```suggestion
Cluster administrators can enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../../virt/about_virt/about-virt.adoc#about-virt[web console].
```

**Review by adellape — COMMENTED** [review]
A few nits.

**Review by adellape — APPROVED** [review]
🔥

**Pull request merged**